examples/authorize-beta/src/main/java/com/dropbox/core/examples/authorize_beta/PkceAuthorize.java

```diff
@@ -0,0 +1,48 @@
+package src.main.java.com.dropbox.core.examples.authorize_beta;
+
+import com.dropbox.core.DbxAuthFinish;
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxPKCEWebAuth;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.DbxWebAuth;
+import com.dropbox.core.TokenAccessType;
+import com.dropbox.core.DbxAppInfo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PkceAuthorize {
+    public DbxAuthFinish authorize(DbxAppInfo appInfo) throws IOException {
+        // Run through Dropbox API authorization process without client secret
+        DbxRequestConfig requestConfig = new DbxRequestConfig("examples-authorize");
+        DbxAppInfo appInfoWithoutSecret = new DbxAppInfo(appInfo.getKey());
+        DbxPKCEWebAuth pkceWebAuth = new DbxPKCEWebAuth(requestConfig, appInfoWithoutSecret);
```

Comment:
PKCE is implicit if you don't send in secret? Do we want that or should we be explicit about enabling PKCE